src/templates/partials/lighttpd.hbs

```diff
@@ -11,36 +17,87 @@ $SERVER["socket"] == ":80" {
 {{/if}}
 }
 
+$HTTP["scheme"] == "https" {
+    # HTTP Strict Transport Security ({{output.hstsMaxAge}} seconds)
+    setenv.add-response-header = (
+        "Strict-Transport-Security" => "max-age={{output.hstsMaxAge}}"
+    )
+}
 {{/if}}
+
+{{#if (minver "1.4.56" form.serverVersion)}}
+# select one TLS module: "mod_openssl" "mod_mbedtls" "mod_gnutls" "mod_wolfssl" "mod_nss"
+#server.modules += ("mod_openssl")
+
+# lighttpd 1.4.56 and later will inherit ssl.* from the global scope if
+# $SERVER["socket"] contains ssl.engine = "enable" and no other ssl.* options
+# (to avoid having to repeat ssl.* directives in both ":443" and "[::]:443")
+$SERVER["socket"] ==     ":443" { ssl.engine = "enable" }
+$SERVER["socket"] == "[::]:443" { ssl.engine = "enable" }
+ssl.privkey = "/path/to/private_key"
+ssl.pemfile = "/path/to/signed_cert_plus_intermediates"
+ {{#if (minver "1.0.2" form.opensslVersion)}}
+  {{#if (minver "1.1.0" form.opensslVersion)}}
+ssl.openssl.ssl-conf-cmd = ("MinProtocol" => {{#if (includes "TLSv1" output.protocols)}}"TLSv1"{{else if (includes "TLSv1.1" output.protocols)}}"TLSv1.1"{{else if (includes "TLSv1.2" output.protocols)}}"TLSv1.2"{{else}}"TLSv1.3"{{/if}})
+  {{else}}
+ssl.openssl.ssl-conf-cmd = ("Protocol" => "ALL, -SSLv2, -SSLv3{{#unless (includes "TLSv1" output.protocols)}}, -TLSv1{{/unless}}{{#unless (includes "TLSv1.1" output.protocols)}}, -TLSv1.1{{/unless}}{{#unless (includes "TLSv1.2" output.protocols)}}, -TLSv1.2{{/unless}}")
+  {{/if}}
+ssl.openssl.ssl-conf-cmd += ("Options" => "{{#if output.serverPreferredOrder}}+{{else}}-{{/if}}ServerPreference")
+   {{#if output.ciphers.length}}
+# TLS modules besides mod_openssl might name ciphers differently
+# See https://redmine.lighttpd.net/projects/lighttpd/wiki/Docs_SSL
+ssl.openssl.ssl-conf-cmd += ("CipherString" => "{{{join output.ciphers ":"}}}")
+   {{/if}}
+# OCSP stapling (input file must be maintained by external script)
+# https://redmine.lighttpd.net/projects/lighttpd/wiki/Docs_SSL#OCSP-Stapling
+#ssl.stapling-file = "/path/to/cert-staple.der"
```

Comment:
You probably want to uncomment this line.
```suggestion
ssl.stapling-file = "/path/to/cert-staple.der"
```

Reply:
No.  I think that is is not appropriate to uncomment a line to a bogus file path.

To enable TLS stapling in lighttpd, the admin must provide (and must regularly update) the file containing stapling DER.  To avoid noise in the error log for a missing file, I think this should remain commented out.

Reply:
It's just that lines that reference bogus file paths are sort of the standard in the SSL config generator. For example the `ssl.privkey`, `ssl.pemfile` and `ssl.ca-file` lines for lighthttpd and most every line in the other servers that points to a file.

I may be misunderstanding what the `ssl.stapling-file` directive is for. How would it differ from those other directives which are set uncommented and point to bogus files?

Reply:
If you do not provide `ssl.pemfile`, lighttpd is unable to make TLS work.

On the other hand, if you do not provide `ssl.stapling-file`, TLS will work, but without TLS stapling.  TLS stapling is optional AND requires additional work by the admin to set it up and maintain.

Reply:
OCSP stapling is a new and optional feature in lighttpd 1.4.56 and later.
Instructions how to maintain `ssl.stapling-file` can be found at [lighttpd TLS OCSP stapling](https://redmine.lighttpd.net/projects/lighttpd/wiki/Docs_SSL#OCSP-Stapling)

Reply:
I modified the template to uncomment `ssl.stapling-file`, but only show it if `form.ocsp` is enabled (the default)